GitUI/Translation/English.xlf

```diff
@@ -8701,6 +8705,14 @@ This will just checkout on the submodule the commit determined by the superproje
       </trans-unit>
     </body>
   </file>
+  <file datatype="plaintext" original="Plugin" source-language="en">
+    <body>
+      <trans-unit id="Description.Text">
+        <source>Plugin Manager</source>
+        <target />
+      </trans-unit>
+    </body>
+  </file>
```

Comment:
Please revert this piece

Reply:
> This is radical, but I think this is a good direction. However now the "Console style" page is pretty empty, and perhaps we could consider moving the console font setting to the "Fonts" page, and the console style dropdown to the "Colors" page (under its own group, like "Console theme"?).
> What do you think?

I could be awayed either way. I like that the console related stuff is in one sub page but I get your thinking as well. 

I was contemplating the fact that the entire ConEmu settings xml can be loaded externally as well and that maybe another feature I could work on next. If we want to try that, then we would need the console sub page to place the xml file path picker for example.

i.e. These are all the settings embedded in that xml file
![image](https://user-images.githubusercontent.com/16128850/124480056-3834c400-ddea-11eb-937c-37d27fb8ab7d.png)

This isn't a reason to keep the console sub page though. Just something that came to my mind. Let me know how you want to proceed. I can go with you suggested approach if you want.

Reply:
> I was contemplating the fact that the entire ConEmu settings xml can be loaded externally as well and that maybe another feature I could work on next. If we want to try that, then we would need the console sub page to place the xml file path picker for example.
> 
> i.e. These are all the settings embedded in that xml file
> ![image](https://user-images.githubusercontent.com/16128850/124480056-3834c400-ddea-11eb-937c-37d27fb8ab7d.png)

Instead of duplicating the ConEmu's settings page, I'd rather see us providing a mechanism to launch the ConEmu configurator. Though personally I don't think it adds any value, you're one of a handful  contributors who expressed the desire to tweak it.

Reply:
Yeah I wouldn't wanna duplicate it. Rather I was thinking let us point to a xml file from our settings. But like you said, it might be overkill.

So provided we want to continue with the reorg or pages, should I tackle the sub page move as a separate PR?